skipper.env

```diff
@@ -137,3 +137,4 @@ NUTANIX_CLUSTER_NAME
 NUTANIX_SUBNET_NAME
 TF_LOG_PATH
 TF_LOG
+TAGS
```

Comment:
Rename it to `CLUSTER_TAGS`, TAGS is just to generic

Reply:
Done